### DISCUSSION TITLE
How to set default role?

### DISCUSSION THREAD

**Discussion by onlineversioncontrolsystem** [TBXark/ChatGPT-Telegram-Workers#250] [Q&A] [answered]
How do I set the default role, so that I don't need to keep asking to change from American spelling to UK spelling?

**Comment by tbxark** [comment, marked as answer]
Set `SYSTEM_INIT_MESSAGE` in the environment variables or use `/setenv SYSTEM_INIT_MESSAGE=your_init_message` in telegram
